### ISSUE TITLE
[Feature Request] Official Templates

### ISSUE THREAD

**Issue by stevefan1999-personal** [mayuki/Cocona#92]
It would be nice if we can just create a Cocona based project just like with ASP.NET Core in Visual Studio startup page, but I have no experience regarding how to make a template like this...

**Comment by Jogai** [comment]
Did you see the samples folder?

**Comment by stevefan1999-personal** [comment]
@Jogai Yes, but it is tedious to setup all the packages every time I make a new project...need to copy and paste a lot of stuff...time is precious and I don't want to do these repetitive stuff often

**Comment by stevefan1999-personal** [comment]
@Jogai I think the link is kinda dead, plus it uses the old style, which is not enough
